zerver/webhooks/gci/tests.py

```diff
@@ -5,6 +5,7 @@ class GoogleCodeInTests(WebhookTestCase):
     STREAM_NAME = "gci"
     URL_TEMPLATE = "/api/v1/external/gci?&api_key={api_key}&stream={stream}"
     WEBHOOK_DIR_NAME = "gci"
+    VIEW_FUNCTION_NAME = "api_gci_webhook"
```

Comment:
Is there a way we can avoid needing to add `VIEW_FUNCTION_NAME`?  

It's OK to need to do it, but it'd certainly be nicer if we could avoid it.

I guess one option is that we could change the coding pattern to always have the view functions have the same name?

Reply:
I guess the other thing we could do is have the `@webhook_view` decorator somehow attach a pointer to the integration to the `zerver/views/integrations.py` data structure, and have the tests look it up there.

Reply:
In our current model, none of the webhooks (including their tests) are aware of the existence of `zerver/lib/integrations.py`. I think the most appropriate thing to do is to lookup the view function with the default naming if `VIEW_FUNCTION_NAME` is not explicitly defined.